src/wrapper/FlatfileClient.ts

```diff
@@ -38,7 +38,7 @@ const environmentSupplier = () => {
 };
 
 const tokenSupplier = () => {
-    const token = CrossEnvConfig.get("FLATFILE_BEARER_TOKEN");
+    const token = CrossEnvConfig.get("FLATFILE_BEARER_TOKEN") || sessionStorage.getItem('FLATFILE_BEARER_TOKEN');
```

Comment:
Pretty sure sessionStorage is only defined in the browser, not node. So I would do `sessionStorage?.getItem(`

Reply:
correct, sounds good!